core/core-themes-support.el

```diff
@@ -39,6 +39,10 @@
   "Face for displaying key bindings in Spacemacs documents."
   :group 'org-faces)
 
+;; FIXME; if a package, such as doom-themes, includes a new theme then a warning
```

Comment:
if so, then there's no need to keep this list around. just check whether the theme is available here

https://github.com/syl20bnr/spacemacs/blob/41ea2b2136caf5b9cce8e7690e43881bb6669a86/core/core-themes-support.el#L315